rp_redcap/tests/test_tasks.py

```diff
@@ -224,17 +229,22 @@ def export_records(
                         "redcap_data_access_group": "my_test_hospital",
                     }
                 ]
-        elif "'2018-01-02'" in filter_logic:
+        elif "'2018-02-01'" in filter_logic:
```

Comment:
I'm wondering if it might be a good idea to use a day date that cannot be confused with a month?
```suggestion
        elif "'2018-02-20'" in filter_logic:
```